### DISCUSSION TITLE
Resizable Sider

### DISCUSSION THREAD

**Discussion by quick691fr** [refinedev/refine#6372] [Integration request] [answered]
Hi,

I see on the ThemedSiderV2 component :

https://github.com/refinedev/refine/blob/1fa7b334540d4aac7108fd47846a73afa1f14762/packages/antd/src/components/themedLayoutV2/sider/index.tsx

many static reference of sider size as 200px expanded and 80px collapsed.

Can you make it configurable via style or prop of component ? Because when you have menu with long titles, it's clipped and difficult to resize via styles.

Thanks

**Comment by alicanerdurmaz** [comment, marked as answer]
Hello @quick691fr,

It would be nice to change the default styles of `<ThemedSiderV2 />`, but it’s not possible right now. You can customize it as you like by [swizzling the component](https://refine.dev/docs/ui-integrations/ant-design/components/themed-layout/#customizing-with-swizzle).
